TaskApp/TaskApp/ViewController.swift

```diff
@@ -20,26 +20,26 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
         myTable.dataSource = self
 
 
-        // Do any additional setup after loading the view.
+        // Do any additional setup after loading the view. //
     }
     
     
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return 10
-    }
+    } //
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
 
-        if indexPath.row > -1 {
+        if indexPath.row > -1 { //
```

Comment:
I'm not sure why you're doing this. There shouldn't be any scenario (that I can see) where indexPath.row would return something less than 0. Is this an abundance of caution, or is there some other reason why you've made this check?